From 9adebbeb58776cf0d93dcc59af2677bdf9c46921 Mon Sep 17 00:00:00 2001
From: Viorel Suman <viorel.suman@nxp.com>
Date: Tue, 4 Sep 2018 13:33:49 +0300
Subject: [PATCH 4543/5242] MLK-19432-2: ARM64: dts: imx8mq-evk: MQ specific
 ak5558 compatible

commit  f98251762653926ee6056725b42d3fe3b2f3c7f5 from
https://source.codeaurora.org/external/imx/linux-imx.git

Use MQ specific ak5558 sound card compatible string in order to
handle properly 1:2 bclk:mclk SAI ratio.

Signed-off-by: Viorel Suman <viorel.suman@nxp.com>
Reviewed-by: Shengjiu Wang <shengjiu.wang@nxp.com>
(cherry picked from commit 20d6d65c330a1560407bc99e0a7f90225ceaf7d8)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
index e4651f4..96c97ff 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
@@ -110,7 +110,7 @@
 	};
 
 	sound-ak5558 {
-		compatible = "fsl,imx-audio-ak5558";
+		compatible = "fsl,imx-audio-ak5558-mq";
 		model = "ak5558-audio";
 		audio-cpu = <&sai5>;
 		audio-codec = <&ak5558>;
-- 
1.7.9.5

